src/sentry/quotas/redis.py

```diff
@@ -126,31 +133,57 @@ def get_value_for_result(result):
             results,
         )
 
+    def get_refunded_quota_key(self, key):
+        return 'r:{}'.format(key)
+
+    def refund(self, project, key=None, timestamp=None):
+        if timestamp is None:
+            timestamp = time()
+
+        quotas = self.get_quotas_with_limits(project, key=key)
+
+        if not quotas:
+            return
+
+        client = self.cluster.get_local_client_for_key(six.text_type(project.organization.pk))
```

Comment:
Use `project.organization_id` otherwise, this is going to yield another query. Especially in the way that it's being called since it's the result of a simple `Project.objects.get()`.

Reply:
In similar vein, it looks like other code in quotas hits `project.organization.id`. I think we should clean all of this to just use `project.organization_id` to prevent hitting outselves with a bunch of extra queries.

L82 of sentry/quotas/redis.py specifically.

Reply:
yeah, `is_rate_limited` is where i got this (line 188), but good point. i'll clean up all of these.